kernel/fmt.cc

```diff
@@ -551,6 +560,23 @@ std::vector<VerilogFmtArg> Fmt::emit_verilog() const
 				break;
 			}
 
+			case FmtPart::HIERNAME: {
+				VerilogFmtArg arg;
+				arg.type = VerilogFmtArg::HIERNAME;
+				args.push_back(arg);
+				fmt.str += '%';
+				if (part.plus)
```

Comment:
Are these modifiers actually valid for `%m`?

Reply:
AFAICT the spec is not clear on this and IIRC there was at least some inconsistency between other implementations. In places where the spec is unclear I think it would make sense to support whatever other commonly used open-source tools like iverilog support as long as that's a reasonable interpretation of the spec (although I don't remember what exactly iverilog does here and in general I'd also be fine with not supporting any modifiers for `%m`).

Reply:
> In places where the spec is unclear I think it would make sense to support whatever other commonly used open-source tools like iverilog support as long as that's a reasonable interpretation of the spec (although I don't remember what exactly iverilog does here and in general I'd also be fine with not supporting any modifiers for %m).

I agree with this stance. I just want to make sure that whichever modifiers we implement here actually match other tools, if we do support them.